pyccel/codegen/printing/fcode.py

```diff
@@ -79,22 +78,78 @@
 __all__ = ["FCodePrinter", "fcode"]
 
 known_functions = {
-    "sin": "sin",
-    "cos": "cos",
-    "tan": "tan",
-    "asin": "asin",
-    "acos": "acos",
-    "atan": "atan",
-    "atan2": "atan2",
-    "sinh": "sinh",
-    "cosh": "cosh",
-    "tanh": "tanh",
-    "log": "log",
-    "exp": "exp",
-    "erf": "erf",
-    "Abs": "abs",
-    "sign": "sign",
-    "conjugate": "conjg"
+    "sign": "sign",       # TODO: move to numpyext
+    "conjugate": "conjg"  # TODO: move to numpyext
+}
+
+numpy_ufunc_to_fortran = {
+    'NumpyAbs'  : 'abs',
+    'NumpyFloor': 'floor',  # TODO: might require special treatment with casting
+    # ---
+    'NumpyExp' : 'exp',
+    'NumpyLog' : 'Log',
+    'NumpySqrt': 'Sqrt',
+    # ---
+    'NumpySin'    : 'sin',
+    'NumpyCos'    : 'cos',
+    'NumpyTan'    : 'tan',
+    'NumpyArcsin' : 'asin',
+    'NumpyArccos' : 'acos',
+    'NumpyArctan' : 'atan',
+    'NumpyArctan2': 'atan2',
+    'NumpySinh'   : 'sinh',
+    'NumpyCosh'   : 'cosh',
+    'NumpyTanh'   : 'tanh',
+    'NumpyArcsinh': 'asinh',
+    'NumpyArccosh': 'acosh',
+    'NumpyArctanh': 'atanh',
+}
+
+math_function_to_fortran = {
+    'MathAcos'   : 'acos',
+    'MathAcosh'  : 'acosh',
+    'MathAsin'   : 'asin',
+    'MathAsinh'  : 'asinh',
+    'MathAtan'   : 'atan',
+    'MathAtan2'  : 'atan2',
+    'MathAtanh'  : 'atanh',
+#    'MathCopysign': '???', # TODO
+    'MathCos'    : 'cos',
+    'MathCosh'   : 'cosh',
+#    'MathDegrees': '???',  # TODO
+    'MathErf'    : 'erf',
+    'MathErfc'   : 'erfc',
+    'MathExp'    : 'exp',
+#    'MathExpm1'  : '???', # TODO
+    'MathFabs'   : 'abs',
+#    'MathFmod'   : '???',  # TODO
+#    'MathFsum'   : '???',  # TODO
+    'MathGamma'  : 'gamma',
+    'MathHypot'  : 'hypot',
+#    'MathLdexp'  : '???',  # TODO
+    'MathLgamma' : 'log_gamma',
+    'MathLog'    : 'log',
+    'MathLog10'  : 'log10',
+#    'MathLog1p'  : '???', # TODO
+#    'MathLog2'   : '???', # TODO
+#    'MathPow'    : '???', # TODO
+#    'MathRadians': '???', # TODO
+    'MathSin'    : 'sin',
+    'MathSinh'   : 'sinh',
+    'MathSqrt'   : 'sqrt',
```

Comment:
As above: since this dictionary entry is not used (`sqrt` is a special case), for the sake of clarity I suggest to comment this line out and add a comment.